awesome-books-with-ES6/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
- [ ] This file is your entry point in the app. Let's maintain the YAGNI principle and remove the other root HTML file that you are no longer using.
- [ ] **[OPTIONAL]** It would be nice to move this file to the root path